### ISSUE TITLE
Have you considered making Firemonkey and all your other addons into separate repos?

### ISSUE THREAD

**Issue by Oneechan69** [erosman/support#643]
This would really help with exposure, especially for Firemonkey, so it can show up in Google and Github searches for things like the monkey addons, and addons for running JS scripts. It could get a lot of repo stars which would help a lot with exposure.

**Comment by erosman** [comment]
Thank you for the suggestion. Let's see how popular the move would be.

**Comment by redactedontop** [comment]
Agreed. If their code would also be in GitHub, it'd be even cooler. I can help you set up an automated workflow with github actions. Also, it'd make contributions a lot easier.

**Comment by SailAway33** [comment]
I like the idea and you have done some great work here. 
Thank You

**Comment by erosman** [comment]
I created a separate [Firemonkey](https://github.com/erosman/firemonkey) repo (under construction).

**Comment by redactedontop** [comment]
LET'S GOOO

**Comment by tilkinsc** [comment]
Don't mean to necro this, but could you point me to the source code for the replace addon so I can audit it? It deals with text and I have privacy concerns.

**Comment by erosman** [comment]
@tilkinsc Replace Code is clear. Users can download the extension from AMO with "Save Link As" and open the zip to see the content.

**Comment by tilkinsc** [comment]
AMO?

**Comment by erosman** [comment]
> AMO?

addons.mozilla.org
